DefaultCiActionResolver.php

```diff
@@ -59,7 +71,15 @@ public function addPossibleRoutes(CiActionResolveEvent $event, &$pathParts, $ind
     public function onActionResolveEvent(CiActionResolveEvent $event)
     {
         $path = $event->getRequest()->getPathInfo();
-        $parts = explode('/', substr($path, 1));
+        $part = (string) substr($path, 1);
+        if ($part === '') {
+            $part = 'home/';
```

Comment:
Why support this `/home`? I think it should be left to application to redirect to appropriate home page.